gradle/maven-deployment.gradle

```diff
@@ -34,10 +34,46 @@ afterEvaluate {
 		group = 'Upload'
 		description = 'Create or update the local Maven repository'
 		configuration = configurations.archives
-		repositories.mavenDeployer {
-			repository(url: "file:" + file("${rootDir}/build/maven-repository"))
-			configurePom(pom)
+
+		publishing {
+			repositories {
+				maven {
+					url = "file:" + file("${rootDir}/build/maven-repository")
```

Comment:
sounds fishy to me `${rootProject.buildDir}` would be something id have expected